pb-crm-ui/src/app/invoicing/components/account-summary-page/account-summary-page.component.ts

```diff
@@ -1,36 +1,26 @@
-import { Component, OnInit } from '@angular/core';
-import {ActivatedRoute, Router} from "@angular/router";
-import {InvoicingRoutes} from "../../invoicing-routing.module";
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {AccountModel} from '@hiddentemple/api-interfaces';
+import {AccountCacheService} from '../../services/account-cache.service';
 
 @Component({
   selector: 'app-account-summary-page',
   template: `
-    <p>
-      account-summary-page works!
-    </p>
-    <button (click)="goToDashboard()">Go To Account Dashboard</button>
-    <button (click)="goToInvoice()">Go To Invoice</button>
+    <div class="container">
+      <app-invoice-table [invoices]="account?.invoices"></app-invoice-table>
+    </div>
   `,
-  styles: [
-  ]
+  styles: []
 })
-export class AccountSummaryPageComponent implements OnInit {
 
+export class AccountSummaryPageComponent implements OnInit {
```

Comment:
As well as this classes name and the file name, which is of form `name-lower-case.component.ts` so something like `account-detail.component.ts`